media-sound/qpwgraph/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>proaudio@gentoo.org</email>
+		<name>Gentoo ProAudio Project</name>
+	</maintainer>
```

Comment:
We are not associated with the Gentoo ProAudio project and they don't maintain packages in this overlay. If you are interested in this package, please consider adding yourself as a `<maintainer type="person">` here as has been done by other people for other packages as well.